### PR TITLE
New package: GovernmentOfThePRC.ZheJiangProvince.ZheZhengDing version 2.21.8

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/198966)

### CODE REVIEW

**Comment by DuckDuckStudio** [comment]
I'm not sure the silent install switch are correct...

**Comment by DuckDuckStudio** [comment]
I am unable to resolve this issue.
image

**Comment by DuckDuckStudio** [comment]
- https://github.com/microsoft/winget-pkgs/pull/198966#issuecomment-2543006627

File: manifests/g/GovernmentOfThePRC/ZheJiangProvince/ZheZhengDing/2.21.8/GovernmentOfThePRC.ZheJiangProvince.ZheZhengDing.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: GovernmentOfThePRC.ZheJiangProvince.ZheZhengDing
+PackageVersion: 2.21.8
+PackageLocale: zh-CN
+Publisher: 杭州市西湖区浙江省数据局（浙江省政务服务管理办公室）
+PublisherUrl: https://www.zj.gov.cn/
```
Comment (DuckDuckStudio) [inline manifests/g/GovernmentOfThePRC/ZheJiangProvince/ZheZhengDing/2.21.8/GovernmentOfThePRC.ZheJiangProvince.ZheZhengDing.locale.zh-CN.yaml:8]:
```suggestion
#PublisherUrl: https://www.zj.gov.cn/
```

image
Can this be waived?

Comment (DuckDuckStudio) [reply manifests/g/GovernmentOfThePRC/ZheJiangProvince/ZheZhengDing/2.21.8/GovernmentOfThePRC.ZheJiangProvince.ZheZhengDing.locale.zh-CN.yaml]: If this url can be waived, I will add it back to the manifest later.

**Comment by DuckDuckStudio** [comment]
About https://github.com/microsoft/winget-pkgs/labels/Validation-Domain , Same reason, the domain names of business units and governmental units do not match.

- https://github.com/microsoft/winget-pkgs/pull/198363#issuecomment-2539399312

**Comment by stephengillie** [comment]
Verify:
image
image

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**